echemdb/data/cv/entry.py

```diff
@@ -175,28 +175,27 @@ def format_title(self, e, which_field, as_sentence=True):
 
     def x(self):
         r"""
-        Return the name of the variable on the x-axis, i.e., `"U"`.
-
-        TODO: Adapt along with https://github.com/echemdb/svgdigitizer/issues/106.
+        Return the name of the variable on the x-axis, i.e., ``"E"``.
 
         EXAMPLES::
 
             >>> entry = Entry.create_examples()[0]
             >>> entry.x()
-            'U'
+            'E'
 
         """
         from astropy import units as u
-        if u.Unit(self.figure_description.potential_scale.unit).is_equivalent('V'):
-            return 'U'
-        else:
-            raise ValueError(f"The variable on the x-axis is not equivalent to 'V'.")
+
+        axis = getattr(self.data_description, 'axes', {})['E']
```

Comment:
This now throws a KeyError when there are no axes and when there is no E axis. In both cases, the error says that there is no E which is what we want here I think.